docs/install/install-gcp-broker.md

```diff
@@ -122,3 +127,69 @@ receive events from it, just like any Knative Eventing
 
 You can find demos of the GCP broker in the
 [examples](../examples/gcpbroker/README.md).
+
+## Debugging
+![GCP Broker](images/GCPBroker.png )
+
+### Control Plane
+
+The GCP Broker controller is responsible for provisioning the decoupling topics and pull subscriptions for GCP Brokers,
+and the retry topics and pull subscriptions for the Triggers.
+* Code: The logic in in [broker.go]( https://github.com/google/knative-gcp/blob/master/pkg/reconciler/broker/broker.go).
+The main function is the [shared main function](https://github.com/google/knative-gcp/blob/master/cmd/controller/main.go)
+with other controllers.
+* [Deployment](https://github.com/google/knative-gcp/blob/master/config/500-controller.yaml): By default it's deployed
+as `controller` in the `cloud-run-events` namespace.
+
+### Data Plane
+
+All GCP Brokers share the following data plane components:
+* Ingress. Ingress accepts events over HTTP/HTTPS and persists events in a Pub/Sub topic specific to each Broker.
+  * Code: [main.go](https://github.com/google/knative-gcp/blob/master/cmd/broker/ingress/main.go)
+  * Deployment: It contains a Service and Deployment, both called `broker-ingress` in the `cloud-run-events` namespace.
+* Fanout. Fanout continously pull events from decouple topics for all Brokers, applies Trigger filters, and sends events
+  to consumers. For failed deliveries, it sends the events to the corresponding retry topic.
+  * Code: [main.go](https://github.com/google/knative-gcp/blob/master/cmd/broker/fanout/main.go)
+  * Deployment: It a deployment called `broker-fanout` in the `cloud-run-events` namespace.
+* Retry. Retry continously resends events that have failed in delivery to the consumers.
+  * Code: [main.go](https://github.com/google/knative-gcp/blob/master/cmd/broker/retry/main.go)
+  * Deployment: It a deployment called `broker-retry` in the `cloud-run-events` namespace.
+
+
+### Common Issues
```

Comment:
Have we already fixed the broker recreation issues? If user delete and then re-create the broker with the same name, user needs to delete the `broker-fanout` pod

Reply:
Yes this has already been fixed.